grpc/proto/space.proto

```diff
@@ -245,12 +252,46 @@ service SpaceApi {
       body: "*"
     };
   }
+
+  rpc ToggleBucketBackup(ToggleBucketBackupRequest) returns (ToggleBucketBackupResponse) {
+    option (google.api.http) = {
+      post: "/v1/backup"
+      body: "*"
+    };
+  }
+
+  rpc GetUsageInfo(GetUsageInfoRequest) returns (GetUsageInfoResponse) {
```

Comment:
This one was had at pending discussion but I added it anyway based on the UI since the implementation detail should be abstracted away by this interface right?